doc/md_docs.go

```diff
@@ -31,15 +31,15 @@ func printOptions(buf *bytes.Buffer, cmd *cobra.Command, name string) error {
 	flags := cmd.NonInheritedFlags()
 	flags.SetOutput(buf)
 	if flags.HasAvailableFlags() {
-		buf.WriteString("### Options\n\n```\n")
+		buf.WriteString("### Flags\n\n```\n")
```

Comment:
Changing `Options` to `Flags` in the markdown docs seems fine. Although this introduces another inconsistency between what we internally call "options" in the docs generation. I.e., look at what we do for yaml docs:

https://github.com/spf13/cobra/blob/8b1eba47616566fc4d258a93da48d5d8741865f0/doc/yaml_docs.go#L43

so this makes me abit hesitant since that struct is a public API and I would rather not change that too

Reply:
Yes, I see there are more inconsistencies there, but I think it would be best to handle those in another branch. Especially when there's a function that may need to change.

Reply:
I also discovered that the inconsistencies are present in the man tree generation as well:
https://github.com/spf13/cobra/blob/a0a6ae020bb3899ff0276067863e50523f897370/doc/man_docs.go#L187-L200